account_invoice_merge_operating_unit/wizard/invoice_merge_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+
+    <record id="view_invoice_merge_ou" model="ir.ui.view">
+        <field name="name">Merger Partner Invoice OU</field>
+        <field name="model">invoice.merge</field>
+        <field name="inherit_id" ref="account_invoice_merge.view_invoice_merge"/>
+        <field name="priority" eval="16"/>
+        <field name="arch" type="xml">
+            <xpath expr="//form/group/p[1]" position="replace">
```

Comment:
Replace is not recommended to be used because if other module inherits the original one may fail if this module is installed. Is it possible to make this invisible and create and add the paragraph after?